security/remote_devices/docker-swarm/create-volumes.sh

```diff
@@ -0,0 +1,12 @@
+mkdir -p /tmp/edgex/secrets/
```

Comment:
Please add the header like copyrights and shebang related.

Reply:
Added